docs/vaahflutter/basics/getting-started.md

```diff
@@ -9,22 +9,34 @@
 
 ## How to run the app
 
-### Step 1. Clone repository
+### Step 1. Clone Repository
 
 To clone the repository, you'll need to run the below command in your terminal. Make sure [git](https://git-scm.com/downloads) is installed in your system.
```

Comment:
@we-prajapati-c001 no need to use the `git` remove it

Reply:
yes sir